src/link/Elf.zig

```diff
@@ -423,6 +437,8 @@ fn updateString(self: *Elf, old_str_off: u32, new_name: []const u8) !u32 {
 }
 
 pub fn populateMissingMetadata(self: *Elf) !void {
+    if (self.llvm_ir_module) |_| return;
```

Comment:
This seems like it should be an assertion since `populateMissingMetadata` is only called from `openPath` where we return early if we're using llvm.

Reply:
Ah yes, thanks for noticing